src/tenant/views.py

```diff
@@ -53,11 +55,53 @@ class TenantCreate(PublicOnlyViewMixin, LoginRequiredMixin, CreateView):
 
     def form_valid(self, form):
         """ Copy the tenant name to the schema_name and the domain_url fields."""
-        tenant = form.save(commit=False)
+        from siteconfig.models import SiteConfig
+
         # TODO: this is duplication of code in admin.py.  Move this into the Tenant model?  Perhaps as a pre-save hook?
-        tenant.schema_name = tenant.name.replace('-', '_')
-        tenant.domain_url = f"{tenant.name}.{Site.objects.get(id=1).domain}"
-        return super().form_valid(form)
+
+        # Because commit=False does not only result in not creating a record at the database.
```

Comment:
Good info explaining the change, but doesn't make sense to have in the code since it won't be referring to anything in the code (in the future, put this in the PR description)

Reply:
not sure, if I need to explain this in PR description at all, I already mentioned `TenantCreate.form_valid` method and what was done there (removal of useless comments, see https://github.com/bytedeck/bytedeck/pull/1447/commits/6f3cee507a24e7762ed6bf69bee29accaac5df3a)